dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/MultContextTests20.java

```diff
@@ -64,7 +77,12 @@ public String getLogMethodName() {
 
     @After
     public void stopOverrideServer() throws Exception {
-        super.stopServer();
+        String methodName = testName.getMethodName();
+        if (methodName != null && methodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
+            super.stopServer(true, "CWWKT0015W");
+        } else {
+            super.stopServer();
```

Comment:
why not `stopServer(true)`

Reply:
super.stopServer() will call super.stopServer(true)